examples/custom.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
this file should be called chi.go or something like that, so it's clear what is being used.

Reply:
Yeah I agree.Let me rename it.

Reply:
I think removing the custom file would me much appropriate,and just leave the initial examples untouched.

Reply:
I think this example is useful.

Reply:
I am wondering how to include the go-chi dependency without polluting our go.mod